map2loop/utils.py

```diff
@@ -0,0 +1,112 @@
+import numpy
+from shapely.geometry import Point
+import beartype
+
+
+@beartype.beartype
+def setup_grid(bounding_box: dict):
+    """
+    Setup the grid for interpolation
+
+    Args:
+        bounding_box
```

Comment:
The structure of the bbox dict may be useful

Reply:
I think a data structure for a bounding box that has a method to return a grid would be good. That way you can have the cell size scaling parameter as an attribute of the bounding box object. It would be similar to the BoundingBox class in LoopStructural.

Reply:
@AngRodrigues Docstring added. 
@lachlangrose I agree with your suggestion. I will implement it later.